content/en/serverless/azure_functions/_index.md

```diff
@@ -56,43 +51,117 @@ This page explains how to collect traces, trace metrics, runtime metrics, and cu
    ```
 
-   **Note**: the environment variable to set JVM options depends on the hosting plan (example, Consumption, Elastic Premium, Dedicated). See [Azure Functions Java developer guide][5] for more details on the appropriate environment variable for your hosting plan.
+   **Note**: the environment variable to set JVM options depends on the hosting plan (example, Consumption, Elastic Premium, Dedicated). See [Azure Functions Java developer guide][4] for more details on the appropriate environment variable for your hosting plan.
+
+3. (Optional) **Enable runtime metrics**. See [Java Runtime Metrics][1].
+
+4. (Optional) **Enable custom metrics**. See [Metric Submission: DogStatsD][2].
+
+[1]: /tracing/metrics/runtime_metrics/?tab=java#environment-variables
+[2]: /metrics/custom_metrics/dogstatsd_metrics_submission/?tab=java
+[3]: https://repo1.maven.org/maven2/com/datadoghq/dd-serverless-compat-java-agent/
+[4]: https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-java?tabs=bash%2Cconsumption#customize-jvm
+{{< /programming-lang >}}
+{{< programming-lang lang="dotnet" >}}
+1. **Install dependencies**. Run the following commands:
+   ```shell
+   dotnet package add Datadog.Azure.Functions
+   ```
+
+   Datadog recommends regularly upgrading to the latest version of `Datadog.AzureFunctions` to ensure you have access to enhancements and bug fixes.
+
+2. **Start the Datadog Serverless Compatibility Layer**. Add the following lines to your main application entry point file:
+   If your Azure Function app uses the Isolated Worker model
+   ```csharp
+   using Datadog.Serverless;
+   ...
+   Datadog.Serverless.CompatibilityLayer.Start();
+   ```
+
+   If your Azure Function app uses the In-Process model
+   ```shell
+   dotnet package add Microsoft.Azure.Functions.Extensions
+   ```
+
+   ```csharp
+   using Datadog.Serverless;
+   using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+
+   [assembly: FunctionsStartup(typeof(MyFunctionApp.Startup))]
+
+   namespace MyFunctionApp
+   {
+      public class Startup : FunctionsStartup
+      {
+         public override void Configure(IFunctionsHostBuilder builder)
+         {
+               Datadog.Serverless.CompatibilityLayer.Start();
+         }
+      }
+   }
+   ```
+
+3. Configure Automatic Instrumentation
+   If your Azure Function app runs on Windows
+   ```
+   CORECLR_ENABLE_PROFILING=1
+
+   CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+
+   CORECLR_PROFILER_PATH_64=
+   C:\home\site\wwwroot\datadog\win-x64\Datadog.Trace.ClrProfiler.Native.dll
+
+   CORECLR_PROFILER_PATH_32=
+   C:\home\site\wwwroot\datadog\win-x86\Datadog.Trace.ClrProfiler.Native.dll
+
+   DD_DOTNET_TRACER_HOME=C:\home\site\wwwroot\datadog
+
+   DD_TRACE_HTTP_CLIENT_EXCLUDED_URL_SUBSTRINGS=monitor.azure, applicationinsights.azure, metadata/instance/compute, admin/host, AzureFunctionsRpcMessages.FunctionRpc
```

Comment:
We'll add this one to the tracer defaults so users don't need to set it manually.

```suggestion
```